### DISCUSSION TITLE
multiple authentifications

### DISCUSSION THREAD

**Discussion by jpchev** [minio/minio#14597] [Q&A] [answered]
is it possible to plug min.io to an identity provider (ex. Keycloak) and still using its internal user management?

Apparently yes, but I don't know if this is something officially supported and foreseen along the product roadmap, could you please confirm?

Thanks

**Comment by harshavardhana** [comment, marked as answer]
Right now it is supported @jpchev

**Reply by jpchev** [reply]
thanks, so it means some users can be on Keycloack, and some others in the min.io internal database, right?

**Reply by harshavardhana** [reply]
that is correct - but Console UI doesn't support multiple different login methods right now. This is going to be supported in future @jpchev

**Reply by jpchev** [reply]
Happy to hear that, thanks
